core/modules/user/user.pages.inc

```diff
@@ -557,6 +557,14 @@ function user_cancel_confirm($account, $timestamp = 0, $hashed_pass = '') {
  */
 function user_page() {
   global $user;
+
+  // To help users upgrading from previous versions of Backdrop where this
```

Comment:
- s/previous/older than 1.9.0/

- Maybe mention the https://github.com/backdrop/backdrop-issues/issues/260 issue number